jsdoc.conf.json

```diff
@@ -27,7 +27,7 @@
     "default": {
       "includeDate": false,
       "outputSourceFiles": true
-  },
+    },
```

Comment:
How'd this get past prettier?

Reply:
No idea. Do you know whether my change is correct or the original was correct?